<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_his_qushitu</class>
 <widget class="QDialog" name="Dlg_his_qushitu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>趋势图</string>
  </property>
  <widget class="QPushButton" name="m_btn_back_fast">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd_fast">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back_end">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>|&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_return">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_para">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>心率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>病人姓名</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_interval">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd_end">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>时间间隔</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_para">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>参数</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
